### PR TITLE
Cannot run Docker node against Lisk Sepolia Testnet

### LINKED ISSUE #13: Cannot run Docker node against Lisk Sepolia Testnet
### Expected behavior

User should be able to run the node against Lisk Sepolia Testnet successfully without any issues.

### Actual behavior

Running the node on Docker against the Lisk Sepolia Testnet fails with the following error:

```sh
sameer@Sameers-MBP ~ % docker logs -f lisk-node-node-1
waiting for the execution client to be ready
fetched public IP is: 46.189.37.234
./op-node-entrypoint: line 47: OP_NODE_ROLLUP_CONFIG: unbound variable
```

### Steps to reproduce

Follow the instructions in the README and start a node against the Lisk Sepolia Testnet.

### Which version(s) does this affect? (Environment, OS, etc...)

`main` branch (commit: https://github.com/LiskHQ/lisk-node/commit/4ae05052ad8d096f6eb1fcab126763a0f0259c80)

### PR DESCRIPTION
**Pull request by sameersubudhi**
### What was the problem?

This PR resolves #13 

### How was it solved?

- [x] Add missing environment variable in `.env.sepolia`
- [x] Apply patch for Lisk Sepolia when building the op-geth Docker image
- [x] Update README with the necessary information

### How was it tested?

Running node against Sepolia and Mainnet.

### CODE REVIEW

File: geth/lisk-sepolia-hotfix.patch [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+<+>UTF-8
+===================================================================
+diff --git a/op-node/rollup/derive/system_config.go b/op-node/rollup/derive/system_config.go
+--- a/op-node/rollup/derive/system_config.go	(revision 7283eb987be7b4413ae5b3088f83aa6a7b83c119)
++++ b/op-node/rollup/derive/system_config.go	(revision 9720ac43f8564a80866c12c7c6fd1e60986ff98e)
+@@ -21,6 +21,7 @@
+ 	SystemConfigUpdateGasConfig         = common.Hash{31: 1}
+ 	SystemConfigUpdateGasLimit          = common.Hash{31: 2}
+ 	SystemConfigUpdateUnsafeBlockSigner = common.Hash{31: 3}
++	SystemConfigUpdateGasConfigEcotone  = common.Hash{31: 4}
```
Comment (shuse2) [inline geth/lisk-sepolia-hotfix.patch]: is this bug reported somewhere?

Comment (sameersubudhi) [reply geth/lisk-sepolia-hotfix.patch]: Gelato was handling this and had shared the patch earlier. We could check with them.

**Review by nagdahimanshu — APPROVED** [review]
Looks good and works as expected 👍

File: geth/lisk-sepolia-hotfix.patch (hunk 1 above)
Comment (sameersubudhi) [reply geth/lisk-sepolia-hotfix.patch]: Updated the comments corresponding the patch to be precise.

**Review by shuse2 — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -244,6 +244,13 @@ Refer to the `reth` configuration [documentation](https://reth.rs/cli/reth/node.
 
 Navigate to your `op-node` directory and start service by running the command:
 
+**Note**:
+
+- Please make sure to patch your `op-node` with [`lisk-hotfix.patch`](./geth/lisk-hotfix.patch) for an unhandled `SystemConfig` event emitted, affecting the Lisk nodes resulting in error logs. This patch is temporary until our RaaS provider updates the `SystemConfig` contract.
```
Comment (nagdahimanshu) [inline README.md:249]: The patch is only needed when running Lisk Sepolia nodes. May be we can update it to
```
Please make sure to patch your `op-node` with [`lisk-hotfix.patch`](./geth/lisk-hotfix.patch) for an unhandled `SystemConfig` event emitted, affecting the Lisk Sepolia node resulting in error logs.
```

Comment (sameersubudhi) [reply README.md:249]: This affects both Mainnet and Sepolia Testnet. Let's discuss this offline.

**Pull request merged**